### PR TITLE
Tech CORE-429 | Change the inspr cmd cluster command to ignore the .inspr/token

### PR DESCRIPTION
**Pull request by luiz-couto**
# Description
* The auth init command now don't generate the not found token error if the token file don't exist.
* The admin user now have all permissions

Kind: Tech Pendency

Section: Auth Client / UIDP Client

Summary:
* Now the cluster init command write a empty token file before send the request, avoiding the not found token error
* Admin user permissions update: now it has all the permissions

## Changelog

* Updated the Init function in the auth package, now it writes a empty token file
* Added the admin permissions in auth models
* When initializing the admin, now using the admin permissions above

### CODE REVIEW

File: pkg/controller/client/auth.go [hunk 1]
```diff
@@ -38,7 +41,13 @@ func (ac *AuthClient) Init(ctx context.Context, key string) (string, error) {
...
 	authDI := auth.JwtDO{}
 
-	err := ac.reqClient.Send(ctx, "/init", "POST", authDO, &authDI)
+	tokenPath := cmd.InsprOptions.Token
+	err := ioutil.WriteFile(tokenPath, []byte(""), os.ModePerm)
+	if err != nil {
+		return "", err
+	}
+
+	err = ac.reqClient.Send(ctx, "/init", "POST", authDO, &authDI)
```
Comment (franciscobonand) [inline pkg/controller/client/auth.go]: Might be a good idea to change "POST" to http.MethodPost.  

Edit: Btw, remember to update CHANGELOG.md

**Review by franciscobonand — CHANGES_REQUESTED** [review]

**Review by nicholaspcr — APPROVED** [review]

File: pkg/controller/client/auth.go (hunk 1 above)
Comment (luiz-couto) [reply pkg/controller/client/auth.go]: Yess

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
 
 # Changelog
 
+### #67 Tech CORE-429 | Change the inspr cmd cluster command to ignore the .inspr/token
+- features:
+    - Updated the Init function in the auth package, now it writes a empty token file
```
Comment (franciscobonand) [inline CHANGELOG.md]: Maybe "Updated the Init function in the auth package so now it creates `.inspr/token` if it doesn't exist and writes the token in the file"

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,12 @@
...
+    - Added the admin permissions in auth models
+    - When initializing the admin, now using the admin permissions above
```
Comment (franciscobonand) [inline CHANGELOG.md]: "The admin user is now initialized with the admin permissions created in auth models"

**Review by franciscobonand — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (luiz-couto) [reply CHANGELOG.md]: That's not what it does, actually. It just creates a empty token file that will be filled later on. But I updated the Changelog to be more clear.

File: CHANGELOG.md (hunk 3 above)
Comment (luiz-couto) [reply CHANGELOG.md]: Okk

**Review by franciscobonand — APPROVED** [review]
LGTM!

**Pull request merged**